components/common/NewProfile.tsx

```
@@ -204,340 +207,339 @@ export const NewProfile = ({
-        <div className="px-4 mt-6">
-          <div className="bg-gray-800/20 backdrop-blur-sm rounded-xl border border-gray-700/30 overflow-hidden">
-            {/* Join Telegram */}
+        {/* SETTINGS GROUP */}
+        <div
+          className="rounded-[35px] border border-white/5 overflow-hidden shadow-xl"
+          style={{ backgroundColor: "rgba(255,255,255,0.02)" }}
+        >
+          <div className="p-2 space-y-1">
             <a
               href="https://t.me/minilendxyz"
               target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)